Gemfile

```diff
@@ -54,7 +54,7 @@ end
 
 group :system_tests do
   gem 'beaker',                       :require => false
-  gem 'beaker-docker',                :require => false
+  gem "beaker-docker", require: false, git: 'https://github.com/treydock/beaker-docker', ref: 'c6d1d3dc5f1e8b7770109793d15cfc9927eb1961'
```

Comment:
This should have a comment such as "# TODO Waiting on link/to/pullrequest to be merged or similar functionality to support EL8"

Reply:
I'll update PR. Looks like puppetlabs un-archived beaker-docker. When I forked the repo it was archived (months ago) and pull requests were not being accepted.  We can use upstream gem.

Reply:
PR updated to remove forked gem